src/platform/user/authentication/components/SessionTimeoutModal.jsx

```diff
@@ -28,8 +28,7 @@ export class SessionTimeoutModal extends React.Component {
   }
 
   componentDidUpdate() {
-    this.serviceName =
-      this.props.serviceName === undefined ? '' : this.props.serviceName;
+    this.serviceName = this.props.serviceName || '';
```

Comment:
🤔  Can we change this?

Move the lookup inside the `mapStateToProps`

```javascript
// in componentDidUpdate
this.serviceName = this.props.serviceName;

// in mapStateToProps
// signInServiceNameSelector for SSOe, sessionStorage for SiS, fallback to empty string
const serviceName = signInServiceNameSelector(state) || sessionStorage.getItem('ci') || '';
return {
  // rest of code
  serviceName
 }
 ```

Reply:
Will do!